src/Libstapsdt.cs

```diff
@@ -6,11 +6,14 @@
 #pragma warning disable CA5392 // Use DefaultDllImportSearchPaths attribute for P/Invokes
                                // ...but it has no effect on non-Windows platforms or the Mono runtime.
 
-namespace Libstapsdt;
+#pragma warning disable CA1401 // P/Invokes should not be visible
```

Comment:
Adding a new project to the solution triggered multiple new violations, hence the changes in this file.

(same behaviour persists with `8.0.401` SDK)